app/src/routes/__root.tsx

```diff
@@ -24,7 +26,9 @@ export const Route = createRootRouteWithContext<{
         <React.Suspense fallback="">
           <TanStackRouterDevtools position="bottom-right" />
         </React.Suspense>
-        <Outlet />
+        <InntektsmeldingSkjemaStateProvider>
+          <Outlet />
+        </InntektsmeldingSkjemaStateProvider>
```

Comment:
Burde denne kanskje ligge på et eget nivå for nye inntektsmeldinger? Vi kan uansett fikse det senere, men til ettertanke